src/energyplus/ForwardTranslator/ForwardTranslateEnergyManagementSystemActuator.cpp

```diff
@@ -65,113 +69,203 @@ namespace openstudio {
+          zoneOrSpaceName = spaces.front().nameString();
+          LOG(Error, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load_->nameString()
+                                  << "' attached to the SpaceType '" << spaceType_->nameString()
+                                  << "' but you have turned on ForwardTranslation's Space Feature. The Space Type has multiple spaces "
+                                     "attached. Falling back to using the first Space '"
+                                  << zoneOrSpaceName << "'.");
+        }
       } else {
-        idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentUniqueName, m.nameString());
-        idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentType, modelObject.actuatedComponentType());
-        idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentControlType, modelObject.actuatedComponentControlType());
-        m_idfObjects.push_back(idfObject);
-        return idfObject;
+        // Not translating Spaces
+        std::set<std::string> zoneNames;
+        for (const auto& space : spaceType_->spaces()) {
+          if (auto tz_ = space.thermalZone()) {
+            zoneNames.insert(tz_->nameString());
+          }
+        }
+        if (zoneNames.empty()) {
+          LOG(Error, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load_->nameString()
+                                  << "' attached to the SpaceType '" << spaceType_->nameString()
+                                  << "' but the Space Type has zero spaces attached to any Thermal Zones, so it will not be translated.");
+          return boost::none;
+        }
+        zoneOrSpaceName = *(zoneNames.begin());
+        if (zoneNames.size() == 1) {
+          LOG(Warn, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load_->nameString() << "' attached to the SpaceType '"
+                                 << spaceType_->nameString()
+                                 << "' but you have turned off ForwardTranslation's Space Feature. Falling back to using the attached ThermalZone '"
+                                 << zoneOrSpaceName << "' since there is only 1.");
+        } else {  // if (zoneNames.size() > 1) {
+          // TODO: do we grab the first space or just don't translate it?
+          LOG(Error, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load_->nameString()
+                                  << "' attached to the SpaceType '" << spaceType_->nameString()
+                                  << "' but you have turned off ForwardTranslation's Space Feature. "
+                                     "Your SpaceType ends up linked to multiples Thermal Zones. Falling back to using the first Thermal Zone '"
+                                  << zoneOrSpaceName << "'.");
+        }
```

Comment:
If we are NOT translating spaces.... We construct a std::set of all the found Thermal Zone Names (so it's sorted and unique)
If we can't find any, throw. Otherwise, grab the first one, issue a warning if one only, an error if two or more